Assets/Treasure/TDK/Runtime/Identity/TDK.Identity.cs

```diff
@@ -93,15 +94,15 @@ await TDKServiceLocator.GetService<TDKThirdwebService>().Wallet.CreateSessionKey
 
         private async Task<List<User.Signer>> GetActiveSigners()
         {
-            var activeSigners = await TDKServiceLocator.GetService<TDKThirdwebService>().Wallet.GetAllActiveSigners();
+            var activeSigners = await TDKServiceLocator.GetService<TDKThirdwebService>().ActiveWallet.GetAllActiveSigners();
             return activeSigners.Select(activeSigner => new User.Signer
             {
-                isAdmin = activeSigner.isAdmin ?? false,
-                signer = activeSigner.signer,
-                approvedTargets = activeSigner.permissions.approvedCallTargets.ToArray(),
-                nativeTokenLimitPerTransaction = activeSigner.permissions.nativeTokenLimitPerTransaction,
-                startTimestamp = activeSigner.permissions.startDate,
-                endTimestamp = activeSigner.permissions.expirationDate
+                isAdmin = false, // TODO activeSigner.isAdmin is not available anymore
```

Comment:
GetAllAdmins/GetAllActiveSigners have been separated and should not overlap https://thirdweb-dev.github.io/thirdweb-dotnet/docs/Thirdweb.SmartWallet.html#Thirdweb_SmartWallet_GetAllAdmins

Reply:
Hmm, @0xFirekeeper does it make sense to change it like [this](https://github.com/TreasureProject/tdk-unity/pull/109/commits/6d013370c3f12f2c7a136db682e6852720aa400e)? 
```cs
var allAdmins = await thirdwebService.ActiveWallet.GetAllAdmins();
// ...
isAdmin = allAdmins.Contains(activeSigner.Signer),
```
Not sure if thats on the right track, im not familiar with how the admins work

Reply:
Don't think you care about admins in this case, you're probably just checking for active signers to see if you need to recreate a session key for the backend wallet

Reply:
`isAdmin` seems to be used [here](https://github.com/TreasureProject/tdk-unity/blob/thirdweb-v5.0.0/Assets/Treasure/TDK/Runtime/Identity/TDK.Identity.cs#L121) to skip some date validations and permission checks. Not sure if this is necessary or not, maybe @alecananian has more context?
```cs
// If this signer is an admin, they always have the required permissions
(signer.isAdmin || (
    // Start date has passed
    signer.startTimestamp < nowDate &&
    // Expiration date meets minimum time requirements
    signer.endTimestamp >= minEndDate &&
    // Expiration date is not too far in the future (10 years because Thirdweb uses this for admins)
    // This check is to prevent sessions from being created with timestamps in milliseconds
    signer.endTimestamp <= maxEndDate &&
    // All requested targets are approved
    requestedCallTargets.All(callTarget => signerApprovedTargets.Contains(callTarget)) &&
    // Native token limit per transaction is approved
    signer.nativeTokenLimitPerTransaction >= nativeTokenLimitPerTransaction
));
```

Reply:
I'd prefer to keep the `isAdmin` check in there in case we make the `backendWallet` an admin for certain partners. If there's a way to parallelize fetching the admins and signers at the same time (in JS we use `Promise.all` for this), then it likely won't add much time to this check